src/libbson/CMakeLists.txt

```diff
@@ -228,7 +228,8 @@ if(ENABLE_STATIC)
 endif()
 
 if(ENABLE_SHARED)
-   add_library(bson_shared SHARED $<$<PLATFORM_ID:Windows>:libbson.rc>)
+   configure_file (libbson.rc.in libbson.rc)
```

Comment:
Suggest guarding this call to `configure_file()` along with the addition of `libbson.rc` under a separate `if(WIN32)` branch for consistency with https://github.com/mongodb/mongo-cxx-driver/pull/1046, rather than using an inline generator expression in the `add_library()` command.